### DISCUSSION TITLE
Using several timers in parallel affecting each other

### DISCUSSION THREAD

**Discussion by Ben-Anton-Utzer** [contrem/arduino-timer#84] [Q&A]
Hi,

I`m implementing two timers in my code. If they run lonely for themselves it works fine. But working with two it seems they are affecting each other. My functions behind the timer need to run through not disturbing them. 

Are they any ways if timer priority handling / locking each other or so? 

BR
Ben

**Comment by contrem** [comment]
Please share some example code, otherwise we're just making guesses.

All timer instances are independent and shouldn't affect each other. Any shared state outside of the timer is another story.
